packages/terra-abstract-modal/src/AbstractModal.jsx

```diff
@@ -141,35 +124,34 @@ class AbstractModal extends React.Component {
     const mainDocumentElement = document.querySelector(this.props.rootSelector);
 
     if (mainDocumentElement) {
-      const inert = +mainDocumentElement.getAttribute('data-overlay-count');
+      const dataOverlayCount = +mainDocumentElement.getAttribute('data-abstract-modal-overlay-count');
 
-      if (inert === 1) {
-        mainDocumentElement.removeAttribute('data-overlay-count');
+      if (dataOverlayCount === 1) {
+        mainDocumentElement.removeAttribute('data-abstract-modal-overlay-count');
         mainDocumentElement.removeAttribute('inert');
-        // Ensures aria-hidden is properly cleaned up
-        setTimeout(() => mainDocumentElement.removeAttribute('aria-hidden'), 0);
-      } else if (inert && inert > 1) {
-        mainDocumentElement.setAttribute('data-overlay-count', `${inert - 1}`);
+      } else if (dataOverlayCount && dataOverlayCount > 1) {
```

Comment:
nothing major, but i believe we can remove the truthy check because `dataOverlayCount` will always be a number. 

setAttribute will return `null` or `''` if the attribute is not found. casting `null` or `''` to a number will return the number `0`

```suggestion
      } else if (dataOverlayCount > 1) {
```

Reply:
Did not know that about getAttribute! Thanks